sw/device/tests/penetrationtests/firmware/ibex_fi.c

```diff
@@ -102,7 +107,8 @@ static dif_flash_ctrl_device_info_t flash_info;
 // we can do the write/read test without the risk of clobbering data
 // used by the program.
 OT_SECTION(".data")
-static volatile uint32_t sram_main_buffer[32];
+
+static volatile uint32_t sram_main_buffer[256];
```

Comment:
As discussed offline: Could you add a small comment, why this window is increased, but only 32 different entries are used later: `(i % 32)` in the code

Reply:
Thanks @m-temp - good point. I've added a description to this.